lib/index.js

```diff
@@ -59,6 +68,14 @@ const setupHarakiri = (options) => {
         terminateWorker(terminatingWorkers.shift());
       }
     }
+    if (closeTimeout && Object.keys(closingWorkers).length > 0) {
```

Comment:
Checking keys length is redundant, for loop will handle empty obj.

Reply:
Fixed